assets/pages/contributions-landing/pagesVersions/horizontalLayoutLandingPage.jsx

```diff
@@ -78,8 +78,8 @@ const CountrySwitcherHeader = CountrySwitcherHeaderContainer(
 
 const HorizontalLayoutLandingPage: (PropTypes) => React.Node = (props: PropTypes) => {
   const annualContributeCopy = 'Make a recurring commitment to support The Guardian long term or a single contribution as and when you feel like it – choose the option that suits you best.';
-  const annualTestVariant = props.store && props.store.getState().common.abParticipations.annualContributionsRoundTwo;
-  const copyText = annualTestVariant === 'annual' || annualTestVariant === 'annualHigherAmounts'
+  const annualTestVariant = props.store && props.store.getState().common.abParticipations.annualContributionsRoundThree;
+  const copyText = annualTestVariant === 'control' || annualTestVariant === 'annualAmountsA'
```

Comment:
again, this ternary operator feels a bit misleading now since if you have knowledge of the test (two variants, 100% of the audience) you know that in practice the condition will always evaluate to true